ouroboros-consensus/src/ouroboros-consensus/Ouroboros/Consensus/MiniProtocol/ChainSync/Client/Jumping.hs

```diff
@@ -886,13 +892,30 @@ promoteToDynamo ::
   ChainSyncClientHandle m blk ->
   STM m ()
 promoteToDynamo peerStates dynId dynamo = do
-  fragment <- csCandidate <$> readTVar (cschState dynamo)
   mJumpInfo <- readTVar (cschJumpInfo dynamo)
-  -- If there is no jump info, the dynamo must be just starting and
-  -- there is no need to set the intersection of the ChainSync server.
-  let dynamoInitState = maybe DynamoStarted DynamoStarting mJumpInfo
-  writeTVar (cschJumping dynamo) $
-    Dynamo dynamoInitState $ pointSlot $ AF.headPoint fragment
+  jumping' <- readTVar (cschJumping dynamo) >>= \case
+    -- An 'Objector' that already 'Started' need not be disrupted.
+    --
+    -- TODO Intuitively, a 'Starting' 'Objector' also need not be disrupted,
+    -- but disrupting it wouldn't waste any @MsgRollForward@s. More concretely,
+    -- it's not obvious to me how to build a 'DynamoStarting' from a
+    -- 'Starting'.
```

Comment:
I agree that there is no need to have special logic for this case, and I think it is fine to "downgrade" this into a NOTE.